README.md

```diff
@@ -147,6 +147,26 @@ unix
 unix-compat
 ```
 
+Versioning scheme
+-----------------
+
+All of the library components in this repository conform to the Haskell [Package
+Versioning Policy][pvp]. Every executable has a corresponding library, and they
+share a version number.
+
+[pvp]: https://pvp.haskell.org/.
+
+The Crux packages (`crux{,-llvm,-mir}`) all share the same version number. The
+major version of Crux is bumped on *every* release, regardless of the changes
+(or lack thereof) in these individual packages.
```

Comment:
It might be worth mentioning that an exception to this versioning rule is `crux`, which is a library-only package that is shared in common between all of the other `crux-*` packages.